core/src/test/java/org/hibernate/ogm/test/type/converter/JpaAttributeConverterGridTypeApplicationTest.java

```diff
@@ -81,11 +79,12 @@ protected void configure(Map<String, Object> cfg) {
 
 	@Override
 	protected Class<?>[] getAnnotatedClasses() {
-		return new Class<?>[] {
+		return new Class<?>[]{
 				Printer.class
 		};
 	}
 
+	@SuppressWarnings("serial")
 	public static class CustomMapDialect extends MapDialect {
```

Comment:
IIRC, we decided to ignore these warnings globally (at least that's what I did in my Eclipse configuration).

This commit is a lot of noise for such a small thing.

Reply:
OK